ansible/playbooks/windows.yml

```diff
@@ -440,10 +472,12 @@
     - name: Install Visual Studio Community 2013
       raw: 'C:\TEMP\vs2013_community.exe /Silent /NoRestart
             /Log C:\TEMP\vs2013_install_log.txt'
+      when: (vs2013_installed.stat.exists == false)
       tags: visual_studio_2013
 
     - name: Reboot machine after Visual Studio installation
       win_reboot:
+      when: (vs2013_installed.stat.exists == false)
```

Comment:
In case of an issue, we don't want the playbook to hang forever. so there should be a timeout on reboot.
example:      
``` 
- name: Reboot machine after Visual Studio 2013 installation
  win_reboot:
    shutdown_timeout: 3600
    reboot_timeout: 3600
  when: (vs2013_installed.stat.exists == false)
  tags: visual_studio_2013
```

Reply:
Fixed, added 30 mins timeout.

Reply:
@bblondin ready for review, thank you!